frontend/src/metabase/nav/containers/CollectionBreadcrumbs/CollectionBreadcrumbs.tsx

```diff
@@ -5,7 +5,12 @@ import type { State } from "metabase-types/store";
 import CollectionBreadcrumbs from "../../components/CollectionBreadcrumbs";
 
 const collectionProps = {
-  id: (state: State) => getCollectionId(state) ?? "root",
+  id: (state: State, props: { collectionId?: string }) => {
```

Comment:
This allows us to provide a `collectionId` to start from, rather than always starting from that of the redux store (which we would want for the SDK)